### PR TITLE
Wagtail and Next.js setup done with tailwind CSS

### PR DESCRIPTION
**Pull request by nandini584**
- Wagtail set up done
- Next.js set up done 

I have done this setup by following the link shown in [Link](https://github.com/wagtail/nextjs-loves-wagtail).
I have used `npx create-next-app@latest` without including typescript and including tailwind css.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+__pycache__
+env
```
Comment (thibaudcolas) [inline .gitignore:2]: Let’s add virtual environment patterns so Python devs have an easier time:

```suggestion
env
venv
.venv
```

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+node_modules/
+/npm-debug.log
+/.idea/
+.vscode
+coverage
+.python-version
+
+# Distribution / packaging
+.Python
+env/
```
Comment (thibaudcolas) [inline .dockerignore:24]:
```suggestion
env/
venv
.venv
```

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+db.sqlite3
+/media
+/static
```
Comment (thibaudcolas) [inline .gitignore:7]: See the last line missing a newline – we’ll need to add a `.editorconfig` to the project.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Use an official Python runtime based on Debian 10 "buster" as a parent image.
+FROM python:3.8.1-slim-buster
```
Comment (thibaudcolas) [inline Dockerfile:2]: Hmm, this is quite out of date! Not something to worry about for now but let’s add this to our backlog.

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Django>=4.2,<4.3
+wagtail>=5.2,<5.3
```
Comment (thibaudcolas) [inline requirements.txt]: Will need upgrading too

**Review by thibaudcolas — APPROVED** [review]
This is looking good to me @nandini584. There are minor changes I’d recommend but I think I’d rather merge this now and make tweaks later.

There are a few things I don’t think we’ll use at all and will remove before merging, so we don’t have them in the `main` git history:

- `search`
- Vercel / Next.js logos
- Welcome page

File: Dockerfile (hunk 4 above)
Comment (nandini584) [reply Dockerfile:2]: okay

File: requirements.txt (hunk 5 above)
Comment (nandini584) [reply requirements.txt]: done

**Pull request merged**